docs/Dockerfile.local

```diff
@@ -0,0 +1,7 @@
+FROM mkdocs-example:0.1.0
+ARG mkdocs_uid
+ARG mkdocs_gid
+
+USER root
+RUN groupdel dialout && groupadd -g "$mkdocs_gid" mkdocs && useradd -u "$mkdocs_uid" -g mkdocs -m mkdocs
```

Comment:
This is macOS-specific (`staff` group, gid 20).

Should avoid doing this on other platforms, and even on macOS it probably is a no-op since thew default userns configuration in most engines forbids you from using uids and gids below 500.